src/media/css/containers/login.styl

```diff
@@ -0,0 +1,20 @@
+@import '../lib'
+
+
+.login-form
+  background $color--white
+  border-radius 5px
+  max-width 400px
```

Comment:
maybe 320px so we're sure it fits on mobile? could also add a var to lib/layout.styl